src/audio/dai-zephyr.c

```diff
@@ -671,6 +671,7 @@ static int dai_capture_params(struct comp_dev *dev, uint32_t period_bytes,
 		prev = dma_block_cfg;
 		prev->next_block = ++dma_block_cfg;
 	}
+
```

Comment:
Hm... looks like the original patch here got fixed upstream, and the merge just left in this whitespace-only delta.  Will remove, obviously.